src/aggregate/expr/expression.c

```diff
@@ -18,7 +18,8 @@ static int evalFunc(ExprEval *eval, const RSFunctionExpr *f, RSValue *result) {
   for (size_t ii = 0; ii < nargs; ii++) {
     args[ii] = (RSValue)RSVALUE_STATIC;
     argspp[ii] = &args[ii];
-    if (evalInternal(eval, f->args->args[ii], &args[ii]) == EXPR_EVAL_ERR) {
+    if (evalInternal(eval, f->args->args[ii], &args[ii]) == EXPR_EVAL_ERR &&
+        eval->err->code != QUERY_ENOPROPVAL) {
```

Comment:
Can you explain why you need it? maybe add a comment,

Reply:
@MeirShpilraien since changing the behavior of `evalProperty` to return OK if a field does not exist would have a wide affect. It also broke predicates such as `@f1 < @f2`.